pom.xml

```diff
@@ -116,32 +116,6 @@
 
     <dependencyManagement>
         <dependencies>
-            <dependency>
```

Comment:
Why not keep these with `${project.version}`. Than you can remove all of the versions in the modules.

Reply:
No particular reason, I think Mathijs recommended this way in another repo with multiple modules. Maybe it is more standard? Anyway I have incooporated the way you suggested as it does seem to be more inline with how we do it elsewhere.

Reply:
Ok, now you can get rid of `<version>${project.version}</version>` in all submodules.

Reply:
It should already be gone, it was included in the commit